src/incubator/Slider/index.tsx

```diff
@@ -0,0 +1,575 @@
+import isNumber from 'lodash/isNumber';
```

Comment:
Run document formatting on the file, it is not formatted

Reply:
Done